hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBTable.java

```diff
@@ -327,6 +327,7 @@ public long estimateNumKeys(Session session) {
 
         @Override
         public byte[] position(String position) {
+            // The value of START AND END are same
```

Comment:
AND => and
but prefer don't assume START and END are the same, prefer `START.equals(position) || END.equals(position)`